tasks/generate.py

```diff
@@ -29,12 +26,12 @@ def announce(ctx, version):
     contributors_text = '\n'.join('* {}'.format(name) for name in sorted(contributors)) + '\n'
     text = template_text.format(version=version, contributors=contributors_text)
 
-    target = Path(__file__).joinpath('../../doc/en/announce/release-{}.rst'.format(version))
+    target = Path(__file__).parent.joinpath('../doc/en/announce/release-{}.rst'.format(version))
```

Comment:
Hmm what do you mean? I did change to `.parent` for both `release-X.Y.Z.rst` and for `index.rst`.

This is actually curious, the first version works on Windows just fine, but breaks on Linux.